trunk/src/lsc/externaldata.py

```diff
@@ -294,6 +295,7 @@ def sdss_swarp(imglist,_telescope='spectral',_ra='',_dec='',output='', objname='
 
        try:
           _mjd = MJDnow(datetime.datetime(int(str(_dayobs)[0:4]),int(str(_dayobs)[4:6]),int(str(_dayobs)[6:8])))
+          _ut = jd2date(_mjd+2400000.5).strftime('%H%M%S')
```

Comment:
Am I right that this is always going to return 00:00:00? The MJD in the previous line is constructed from a datetime of the form YYYYMMDD, so I don't think there is any time information.

Reply:
Also does this get used anywhere? I think this is inside the SDSS section.

Reply:
Megan says:
> To address the comment about _ut time, the line _ut = jd2date(_mjd+2400000.5).strftime('%H%M%S') does not just return 00:00:00. The database saves the PS1 templates with non-zero UT's. We also made this change because we receive an error when an explicit line defining _ut wasn't defined; thus we added it to the SDSS part as well for completeness' sake.

There's a couple things here:

First of all, I am asking about `_ut` always equaling 00:00:00 for the *SDSS* case, not PS1. I'm pretty sure this will make all our SDSS templates have UTSTART=00:00:00.

Second, the error you're talking about where `_ut` is not defined is because of the code you added below where you introduce UT and UTSTART has header keywords:
```
    hd['UTSTART']  = (_ut,        'universal time')
    hd['UT']       = (_ut,        'universal time')
```
This seems not necessary because we never had it for SDSS before. Would everything run if you just removed all this UT stuff? If the choice is between including UT for PS1 but setting it to the wrong value for PS1, or not including it at all, I think I'd rather not include it.

Reply:
I moved this line outside the conditional so that the UT is calculated from the MJD for both SDSS and PS1. I also removed the keyword `UT` from the header because `UTSTART` is the standard keyword for LCO data.